modules/core/src/math/random-generator/vpGaussRand.cpp

```diff
@@ -48,25 +48,22 @@
 */
 double vpGaussRand::gaussianDraw()
 {
-  static bool AlreadyDone = false;
-  static double x2;
```

Comment:
These static variables store a global state that prevent to have repeatable results.
```